src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj

```diff
@@ -22,6 +22,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <IncludeSymbols>true</IncludeSymbols>
     <Platforms>x64</Platforms>
+    <Version>3.2.2</Version>
```

Comment:
please don't include both 'VersionPrefix' and 'Version' this breaks the build pipe.